### PR TITLE
New options -t, -u, + temp removal

### PR DESCRIPTION
**Pull request by j-mracek**
-t    Is used tun ddt only with listed tests
-u   Change defaults coper repo

### CODE REVIEW

**Comment by j-mracek** [comment]
ps: in dnf-nightly there is still rpm-99-0 version. Please use version 4.99-0 instead for rpm.
Thanks and remove the rpm-99-0 version please.

**Comment by mluscon** [comment]
I think we should get rid of hard-coded dnf-nightly repo and make it optional only. Any opinions on this?

**Comment by jsilhan** [comment]
@ignatenkobrain is working on it. To dnf-nightly we should push successful builds only from all chained jobs.

**Comment by j-mracek** [comment]
DDT - is now very flexible.
The last commit (Allow run test updated from different copr repo) allows to use any Coper repo for docker update. Repo to install tested package is chosen by PREJECT name (``--copr PROJECT BUILDER``). Therefore it is now very flexible.

**Comment by mluscon** [comment]
But you will have to change the hard-coded value every time we change the default repo.

**Comment by ignatenkobrain** [comment]
@j-mracek we are going to remove all building & other stuff except from functional tests themselves from here.

**Comment by j-mracek** [comment]
@jsilhan Please can we merge this PR, and consequently the second one (after I rebase it). I would like to add some more work, but it is impossible at this stage. The functional test with commits in this PR should handle dnf-1.x with dnf-2.0. It is really important. How anyone can write new tests if syntax of tests steps will change radically by installroot PR.

**Comment by j-mracek** [comment]
@ignatenkobrain I was talking about DDT not about whole ci-dnf-stack

File: cidnfstack.py [hunk 1]
```diff
@@ -650,15 +655,13 @@ def _start_commandline():  # pylint: disable=R0912,R0915
...
-        '-c', '--copr', type=unicode, nargs='+', default=['local-build', 'tito'], metavar=('PROJECT', 'BUILDER'),
-        help='the name of the Copr project. If option not present default values are used: "local-build" "tito" and '
-             'the test runs locally (no copr build). '
-             'If BUILDER tito, the "tito" executable must be available. '
-             'If BUILDER librepo, the "cp", "dirname", "echo", "git", "mv", "rpmbuild", "rm", "sed", "sh" and "xz" '
-             'executables must be available.'
-             'If BUILDER libscomps, the "python" and "rpmbuild" executables must be available.')
+        '-n', '--no-dnf-docker-test', help='do not start the functional test of project after Copr build',
+        action="store_true")
     buildparser.add_argument(
```
Comment (mluscon) [inline cidnfstack.py]: Why did you switch -n and -c here?

Comment (j-mracek) [inline cidnfstack.py]: I just alphabetically order options.

Comment (mluscon) [inline cidnfstack.py]: It should be in a separate commit then.

Comment (ignatenkobrain) [inline cidnfstack.py]: it doesn't matter from my POV

Comment (j-mracek) [inline cidnfstack.py]: Sorry I was inaccurate - I added -t option and reorder arguments. I can split it into two. I really want to merge both PRs.

**Comment by ignatenkobrain** [comment]
I have nothing against, +1 from me.

**Comment by mluscon** [comment]
Ok, go ahead @j-mracek.

**Comment by j-mracek** [comment]
Closed

**Comment by j-mracek** [comment]
Merged
